articles/resources/relnotes.md

```diff
@@ -17,6 +17,24 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.12.20092501
+
+*Release date: September 29th, 2020*
+
+This release contains the following:
+
+- Introduction of [Quantum Intermediate Representation (QIR)](https://github.com/microsoft/qsharp-language/tree/main/Specifications/QIR#quantum-intermediate-representation-qir) intended as a common format across different front- and backends; see our [blog](https://devblogs.microsoft.com/qsharp/introducing-quantum-intermediate-representation-qir)
```

Comment:
@bettinaheim Is this correct -- does this release actually contain QIR? I thought it was still in a feature branch.

Reply:
Ah, that is a good point and bad phrasing from my part. No, the implementation is not part of the release; only the spec (and the code) has been made public as part of the language repo. 
```suggestion
- Announcement and draft specification of [Quantum Intermediate Representation (QIR)](https://github.com/microsoft/qsharp-language/tree/main/Specifications/QIR#quantum-intermediate-representation-qir) intended as a common format across different front- and backends; see our [blog](https://devblogs.microsoft.com/qsharp/introducing-quantum-intermediate-representation-qir)
```

Reply:
Yes, I think that's more clear! Thanks.